openqa-bats-review

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python3
+"""
+This script fetches previous cloned BATS jobs to check whether we can
```

Comment:
No, I'm not afraid to ask: What are "BATS jobs"? These descriptions shouldn't use cryptic abbreviations without further explanation.

I was checking https://en.wikipedia.org/wiki/BAT but https://en.wikipedia.org/wiki/BATS but I guess the "BATS" you mean isn't on any of those lists.